testing/example_scripts/fixtures/fill_fixtures/class_variable_auto_use.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+
+class Test(object):
+    @classmethod
```

Comment:
this is the order in-correctness i was talking about in #3780  - we structurally **have** to loose something as long as we wrap